opencog/cython/opencog/Cast.cc

```diff
@@ -8,3 +8,4 @@ using namespace opencog;
 Handle atom_from_the_void(long p) { return *((Handle*) p); }
 long void_from_candle(const Handle& hp) { return (long) (&hp); }
 long void_from_cptr(Handle* hp) { return (long) (hp); }
+long avoid_from_cptr(Handle* hp) { return (long) (hp->operator->()); }
```

Comment:
Amen, there seems clear value for Handles in a distributed system (multiple machines, or multiple processes on the same machine) -- they allow an Atom on one machine to refer to an Atom on another machine... smart pointers don't do that